jdaviz/configs/default/plugins/viewers.py

```diff
@@ -198,7 +198,11 @@ def _expected_subset_layer_default(self, layer_state):
             return
 
         # default visibility based on the visibility of the "parent" data layer
-        layer_state.visible = self._get_layer(layer_state.layer.data.label).visible
+        if self.__class__.__name__ != 'RampvizProfileView':
```

Comment:
can this not do an isinstance check because of circular import?

Reply:
pretty sure, yep.